usaspending_api/api_contracts/contracts/v2/search/spending_over_time.md

```diff
@@ -0,0 +1,160 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending Over Time [/api/v2/search/spending_over_time/]
+
+This endpoint supports the advanced search page and allow for complex filtering for specific subsets of spending data.
+
+## List aggregated award amounts by time period [POST]
+
+This endpoint returns a list of aggregated award amounts grouped by time period in ascending order (earliest to most recent).
+
++ Request (application/json)
+    + Attributes (object)
+        + group: `quarter` (required, enum[string])
+            + Members
+                + `fiscal_year`
+                + `quarter`
+                + `month`
+        + filters (required, FilterObject)
+        + subawards (optional, boolean)
+            True to group by sub-awards instead of prime awards. Defaults to false.
+            + Default: false
+    + Body
+        
+            { 
+                "group": "fiscal_year", 
+                "filters": { 
+                    "keywords": ["Filter is required"] 
+                }
+            }
+
++ Response 200 (application/json)
+    + Attributes
+        + group (required, enum[string])
+            + Members
+                + `fiscal_year`
+                + `quarter`
+                + `month`
+        + results (array[TimeResult], fixed-type)
+
+# Data Structures
+
+## TimeResult (object)
++ `time_period` (required, TimePeriodGroup)
++ `aggregated_amount` (required, number)
+    The aggregate award amount for this time period and the given filters.
+
+## TimePeriodGroup (object)
++ `fiscal_year`: `2018` (required, string)
++ `quarter`: `1` (optional, string)
+    Excluded when grouping by `fiscal_year` or `month`.
++ `month`: `1` (optional, string)
+    Excluded when grouping by `fiscal_year` or `quarter`.
+
+
+## Filter Objects
+### FilterObject (object)
++ `keywords` : poptarts (optional, array[string])
```

Comment:
```suggestion
+ `keywords` : `transport` (optional, array[string])
```
Ross has said that he's okay with "silly" searches so long as they return results, but it might be best to avoid brand names just in case.